config.env

```diff
@@ -0,0 +1,2 @@
+MOD_NAMES=MoreDescriptionForTraits
+MOD_WORKSHOP_IDS=2685168362
```

Comment:
I don't think a `config.env` is really needed.  We can achieve the same functionality with the `docker-compose.yaml`, or adding the extra environment args to the plain Docker command.

Reply:
I agree, I should have removed this upon finishing testing